CMakeLists.txt

```diff
@@ -37,12 +37,17 @@ set(SHARED_UTIL_SRC_FOLDER ${CMAKE_CURRENT_LIST_DIR}/src CACHE INTERNAL "this is
 set(SHARED_UTIL_ADAPTER_FOLDER "${CMAKE_CURRENT_LIST_DIR}/adapters" CACHE INTERNAL "this is where the adapters live" FORCE)
 
 include_directories(${SHARED_UTIL_INC_FOLDER})
-
+get_property(dirs DIRECTORY ${CMAKE_CURRENT_SOURCE_DIR} PROPERTY INCLUDE_DIRECTORIES)
+foreach(dir ${dirs})
+  message(STATUS "dir='${dir}'")
+endforeach()
 CHECK_INCLUDE_FILES(stdint.h HAVE_STDINT_H)
 CHECK_INCLUDE_FILES(stdbool.h HAVE_STDBOOL_H)
 
-if ((NOT HAVE_STDINT_H) OR (NOT HAVE_STDBOOL_H))
+if (WINDOWS)
```

Comment:
I'm interested to understand in what circumstances you have encountered issues with this code. I have cross compiled the SDK for many different targets and, other than one exception, cmake has always been able to find these two headers and hence did not execute the Windows CE circumvention. What platform are you seeing this as an issue?